#!/usr/bin/env python
PACKAGE = "pses_kinect_utilities"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kernel_size", int_t, 0, "Set the size of the kernel of the median filter. Use only even values.", 5, 1, 10)


exit(gen.generate(PACKAGE, "median_filter", "MedianFilter"))
